### PR TITLE
Add projective plane, resolve #893

### LINKED ISSUE #893: Add projective plane
> Great suggestion; I've been meaning to add this for a while since it would give an example of a nontrivial [homogeneous space with the fixed point property](https://topology.pi-base.org/spaces?q=Homogeneous+%2B+Fixed+point+property). (The real projective plane would be another nice one to add, but that's worthy of a separate issue.)

_Originally posted by @danflapjax in https://github.com/pi-base/data/issues/760#issuecomment-2344394836_

### PR DESCRIPTION
**Pull request by yhx-12243**
Closes #893

### CODE REVIEW

File: spaces/S000168/README.md
Comment (yhx-12243) [inline spaces/S000168/README.md:10]: more rich description are WANTED

File: spaces/S000168/properties/P000086.md
Comment (david20000813) [inline spaces/S000168/properties/P000086.md:7]:
```suggestion
For any two points on {S169}, there exists a rotation of $\mathbb R^3$ sending one to the other and preserving all antipodal point pairs.
```

File: spaces/S000168/properties/P000086.md
Comment (david20000813) [inline spaces/S000168/properties/P000086.md:8]:
```suggestion
So this rotation induces a homeomorphism of {S168}.
```

File: spaces/S000168/properties/P000123.md
Comment (david20000813) [inline spaces/S000168/properties/P000123.md:7]: Perhaps it will be clearer to say every point in the sphere (instead of in the projective plane) has such a small neighborhood and to say the quotient map restricts to a homeomorphism from the said neighborhood to an open set in the projective plane?

**Comment by david20000813** [comment]
I think everything looks fine now. But since it seems people are still trying to write a richer description, I'll hold off on approving and merging until that's done, or others can do the approving and merging when the description rewrite is complete and they get to it before I do.

File: spaces/S000168/README.md
Comment (StevenClontz) [inline spaces/S000168/README.md:10]:
```suggestion
The quotient of a {S169} $S^2$ identifying its antipodal points.

Equivalently, the quotient of $[0,1]^2$ identifying $\langle t,0\rangle$ with $\langle 1-t,1\rangle$ and identifying $\langle 0,t\rangle$ with $\langle 1,1-t\rangle$.
```

The latter characterization will be good to compare with Mobius strip, Klein bottle, and torus.

Comment (prabau) [reply spaces/S000168/README.md:10]: Would a representation as a quotient of $[-1,1]^2$ be easier?  Then the identification is just "taking the negative" ($(t,1)$ identified with $(-t,-1)$, etc).  But then it's basically the two dimensional disk slightly deformed with opposite points on the circumference identified.  That seems a cleaner and nicer formulation, and actually one of the presentations suggested by @GeoffreySangston.

**Comment by prabau** [comment]
Making this a visible comment instead of buried as a reply to a suggestion

> Would a representation as a quotient of $[-1,1]^2$ be easier?  Then the identification is just "taking the negative" ($(t,1)$ identified with $(-t,-1)$, etc).  But then it's basically the two dimensional disk slightly deformed with opposite points on the circumference identified.  That seems a cleaner and nicer formulation, and actually one of the presentations suggested by @GeoffreySangston.

File: spaces/S000168/README.md
Comment (GeoffreySangston) [reply spaces/S000168/README.md:10]: Lee's Topological Manifolds Proposition 6.2 has a slick formulation of the square version: 
- $\\{(x, y) : |x| + |y| \le 1\\}/\sim$, where $\sim$ is the minimal equivalence relation such that $(x,y) \sim (-x, -y)$ when $|x| + |y| = 1$.

Though if that seems gimmicky, it could instead be:

- $[-1,1]^2/\sim$, where $\sim$ is the minimal equivalence relation such that $p \sim -p$ for boundary points $p$.

It seems desirable to include the Grassmannian version (maybe just calling it the space of lines through the origin), since that's the definition someone might have in mind, and also the homogeneous coordinates version for the same reason. I think the hemi-sphere is fine to omit, and either the square or disk could be omitted.

Comment (StevenClontz) [reply spaces/S000168/README.md:10]: I am fine with using $[-1,1]^2$ in place of $[0,1]^2$.

Comment (yhx-12243) [reply spaces/S000168/README.md:10]: > The latter characterization will be good to compare with Möbius strip, Klein bottle, and torus.

Nice. I'll consider adding Möbius strip and Klein bottle, etc.

For torus, nowadays all the properties in pi-base cannot distinguish it from S170 (Circle).

**Comment by StevenClontz** [comment]
@austinmohr was probably the first major external (e.g. not a grad student at Auburn) contributor to pi-Base, and used it in his classroom several years back. (This is why several spaces used to include proofs that their topologies were topologies, although we've made the editorial decision to remove those.) The next time I teach topology, I plan to redesign the course (and my notes at https://stevenclontz.github.io/intro-to-topology/frontmatter-1.html ) to use pi-Base as a primary source.

It's worth discussing what an appropriate mission statement for pi-Base would be. My opinion is that pi-Base should be a tool that democratizes and broadens access to our collective knowledge of topology, and that our audience can and should include everyone from students to researchers. Of course, the interests of our volunteer contributors will ultimately determine which areas are better built out.

**Comment by yhx-12243** [comment]
Anyway review this? Seems there are a few days with no comments.

**Comment by GeoffreySangston** [comment]
I'm personally a little bit confused as to which representatives we wanted to include for RP2 in its description. I didn't review because I thought maybe you were going to make a change to that.

I made the list above not to demand any particular one get included, but to have a list which would be easy for us to look at when deciding which representatives should be included.

My personal vote for the representatives to include is:
- Antipodal map definition as primary (easiest to prove topological properties with)
- Homogeneous coordinates definition (one of the definitions people are likely to have in mind when looking for this space in pi-base)
- 1 dimensional subspaces definition (the other definition people might have in mind)
- Unit square gluing definition (either in the form @StevenClontz suggested, or in the form I suggested which I do think is a bit slicker, though @StevenClontz's form is maybe more along the lines of how someone thinks when drawing the picture)

The unit square / disk version actually isn't necessary however, which I think was part of @prabau's trepidation, unless we decide to include properties for which the trait-file argument benefits from it somehow. Linking to the Wikipedia page does already show any interested explorer that model pretty quickly as well. The only reason I vote for the other two is just to make the space recognizable.

**Comment by david20000813** [comment]
Everything looks fine to me. But since I don't have any preference for which representatives of $\mathbb{R}P^2$ to use, I think it's better to let others, who have opinions on this matter, to do the approving and merging.

**Comment by yhx-12243** [comment]
I have no opinion of this, so I think we'll let you edit this description part, because you can edit it anyway. @GeoffreySangston

**Comment by prabau** [comment]
I'd like to see 1,2,3.  Up to others to support 4 if they want to.

File: spaces/S000168/README.md
Comment (GeoffreySangston) [inline spaces/S000168/README.md:6]: I don't think the alias $P_2(\mathbb R)$ is actually very common. Feel free to disagree.

Comment (yhx-12243) [reply spaces/S000168/README.md:6]: Because I see Wikipedia use it. There is also a notation namely $P(\mathbb R^2)$ as you committed.

Comment (yhx-12243) [reply spaces/S000168/README.md:6]: > I don't think the alias P 2 ( R ) is actually very common. Feel free to disagree. P 2 or P 2 perhaps is though.

$P^2$ (`P^2`) is also very reasonable with only defect not specifying which field is ($\mathbb R$ or $\mathbb C$).

I don't agree $\mathbb P^2$ (`\mathbb P^2`) because I think it is nonsense to wrap $P$ with a blackboard (`\mathbb`).

Comment (GeoffreySangston) [reply spaces/S000168/README.md:6]: Maybe the notation $\mathbb{P}^2(\mathbb{R})$ occurs in algebraic geometry to refer to the real points of the algbraic variety $\mathbb{P}^2$? I have very weak familiarity with these things though. I tried for a few weeks last summer to learn about linear algebraic groups to varying success...

File: spaces/S000168/properties/P000089.md
Comment (GeoffreySangston) [inline spaces/S000168/properties/P000089.md:12]: These two posts have the loose end that they assume the reader is familiar with the fact that the identity and antipodal map of $S^2$ are not homotopic. Should we fill in that gap with another reference? The way I know involves degree theory. For that we could cite Corollary 6.12. of Bredon's book *Topology and Geometry*.

**Comment by prabau** [comment]
See https://en.wikipedia.org/wiki/Real_projective_space, Hatcher, etc.

I think the notation $\mathbb RP^n$, $\mathbb CP^n$, etc would be universally recognized, even out of context.  Whereas $P_2(\mathbb R)$, etc would depend on context (e.g. a book denotes it this way at the beginning and then uses that throughout).

So I would change the main name of the space to "Real projective plane $\mathbb RP^2$ " .

And remove the other alias. I don't see a problem with not listing all these possibly aliases that someone used somewhere.  It's really not necessary to allow users to find the space.  Anyone interested in the space will look for "projective plane" or something like it, and I'd say that's good enough.  Think of who the audience will be and what do they need to use pi-base for.

File: spaces/S000168/README.md
Comment (prabau) [inline spaces/S000168/README.md:9]: Need the underscores to match https://en.wikipedia.org/wiki/Real_projective_space (Steven's recommendation).

File: spaces/S000168/README.md
Comment (GeoffreySangston) [reply spaces/S000168/README.md:6]: > I don't agree P 2 (`\mathbb P^2`) because I think it is nonsense to wrap P with a blackboard (`\mathbb`).

I don't think it's nonsense. It is used in Lee's Topological manifolds (which I added as a reference). However, I also prefer it without the blackboard bold.

**Comment by GeoffreySangston** [comment]
I'm unsure about whether we should modify the fixed point property trait file or not. But other than that I think it's ready for someone to review. (I would do it but I have to run! Tomorrow I can if nobody else does.)

File: spaces/S000168/properties/P000089.md
Comment (yhx-12243) [reply spaces/S000168/properties/P000089.md:12]: Maybe hatcher {{zb:1044.55001}} is enough, Section 2.2.

**Comment by yhx-12243** [comment]
> I'm unsure about whether we should modify the fixed point property trait file or not. But other than that I think it's ready for someone to review. (I would do it but I have to run! Tomorrow I can if nobody else does.)

I think we can add the Add the first part of Section 2.2 in Hatcher *if need*.

**Comment by GeoffreySangston** [comment]
I know I said I could review this today. On second thought though, since I've now added commits directly to the PR, I think it's better that a third person like @prabau or @StevenClontz or @david20000813 does the review.

File: spaces/S000168/README.md
Comment (StevenClontz) [inline spaces/S000168/README.md:5]:
```suggestion
name: Real projective plane $\mathbb RP^2$
aliases:
```

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**
